cmd/zqd/listen/command.go

```diff
@@ -23,28 +26,72 @@ The listen command launches a process to listen on the provided interface and
 	New: New,
 }
 
+// defaultLogger ignores output from the access logger.
+var defaultLogger = []logger.Config{
+	{
+		Name:  "zqd",
+		Path:  "stderr",
+		Level: zap.InfoLevel,
+	},
+}
```

Comment:
@alfred-landrum as we discussed elsewhere, I added a default logger which trims out the noisy access logs.